src/main/java/frc/robot/subsystems/endgame/EndgameSubsystem.java

```diff
@@ -33,18 +33,22 @@ public EndgameSubsystem(EndgameIO endgameIO) {
     public void setAction(EndgameAction action) {
         switch (action) {
             case GO_UP:
-                endgameIo.setVolts(4.0);
+                endgameIo.setPosition(1.0);
```

Comment:
target position is in meters so 1.0 will be too much - we should also make this a constant in Constants.java